lib/scheduler.pm

```diff
@@ -24,14 +24,17 @@ use warnings;
 use File::Basename;
 use testapi qw(get_var set_var diag);
 use main_common 'loadtest';
-use YAML::Tiny;
+use YAML::PP;
 use Data::Dumper;
 
 our @EXPORT = qw(load_yaml_schedule get_test_suite_data);
 
 my $test_suite_data;
 my $include_tag = '$include';
 
+my $ypp = YAML::PP->new();
```

Comment:
Just a note:
If we planned to enable merge keys already in this PR, the object would have to be initialized like that:
```perl
my $ypp = YAML::PP->new( schema => [qw(+ Merge )] );
```

Reply:
Thanks for the hint, I will add this note to https://progress.opensuse.org/issues/66206 where we plan to address it and use this functionality to get rid of our self-written solution.